lib/rules/prefer-object-has-own.js

```diff
@@ -87,6 +111,13 @@ module.exports = {
                                 return null;
                             }
 
+                            const tokenJustBeforeNode = sourceCode.getTokenBefore(node, { includeComments: true });
```

Comment:
This example:

```js
function foo() {
    return({}.hasOwnProperty.call)(obj, prop);
}
```

would be autofixed with an unnecessary space:

```js
function foo() {
    return( Object.hasOwn)(obj, prop);
}
```

We should get token before the node that is going to be replaced, which is `node.callee`.

```suggestion
                            const tokenJustBeforeNode = sourceCode.getTokenBefore(node.callee, { includeComments: true });
```